tests/manage/z_cluster/nodes/test_toleration.py

```diff
@@ -7,7 +7,11 @@
     check_toleration_on_pods,
     wait_for_pods_to_be_running,
 )
-from ocs_ci.ocs.node import taint_nodes, untaint_nodes, get_ocs_nodes
+from ocs_ci.ocs.node import (
+    get_ocs_nodes,
+    taint_nodes,
+    untaint_nodes,
```

Comment:
I think it's not used anywhere

Reply:
`untaint_nodes` is done as part of the finalizer.